### ISSUE TITLE
Quoting the license text in CI logs

### ISSUE THREAD

**Issue by tplive** [android-actions/setup-android#384]
Is it a requirement somewhere that the actual license text is quoted in the logs every single time the licenses are accepted? We have a job with 20+ parallel flavours, and this adds almost 800 lines of boilerplate in the logs, for every single flavour on every single run. Not an error, but it seems excessive and these logs go to storage and add cost.

**Comment by ViliusSutkus89** [comment]
Reasonable concern. That whole license agreement is 100 kb of text. However, due to legal reasons, I am not willing to hide the license text by default. We could add an optional param to the action:

```yaml
- name: Setup Android SDK
  uses: android-actions/setup-android@v3
  with:
    log-android-sdk-licenses: true (default) / false
```

This would be pretty straightforward to implement, a perfect starter issue for a rising GitHub Actions hacker, any takers? :)

**Comment by ViliusSutkus89** [comment]
Implemented in v3.1.0. Enjoy
